playwright/integration/auth.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from "../support/test-utils";
+import {
+  API_URL,
+  FULFILL_PATH,
+  LIMITED_ACCESS_WORK_PATH,
+  NYPL_LOGIN_URL,
+} from "~/mocks/mockEnv";
+import { server } from "~/mocks/server";
+
+test.beforeEach(async ({ context }) => {
+  await context.clearCookies();
+  server.listen({
+    onUnhandledRequest: "bypass",
+  });
```

Comment:
I am curious why we need to initialize this when we already do that in the `_app.tsx` file.

Reply:
This is not needed here, it was added when I was initially implementing this. Nice catch

Reply:
Gotcha! I think if we remove this, then we might need to also remove the `server.close()` and `server.resetHandlers` lines below, right? otherwise new tests will start without msw.

I think you wanted to entirely rely on the MSW server when running the app for the tests so it should always stay on. Unless I am not understanding this correctly.

Reply:
A new server is started for each test using the `port` fixture, so resetting the handlers ensures the data is clean.  Closing the server after all tests are run is to clean the test env.